content/en/about/board/2024-03-28.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: page
+show_meta: false
+title: '2024-02-29'
+---
+
+### Date and Time of Meeting
+
+* Call to order: 2024-03-28, 14:09 CET
+* Adjourned: 14:52 CET
+* Next Meeting: 2024-05-02, 07:00 CET
+
+### Roll call
+
+#### Directors and Officers Present
+
+* Daniel Izquierdo-Cortazar (President)
+* Sebastian Spier (Assistant Secretary, Vice President)
+* Yuki Hattori
+* Georg Gruetter (Treasurer)
```

Comment:
```suggestion
* Georg Gruetter (Assistant Treasurer)
```

I believe @tsadler1988 had two roles (Secretary and Treasurer) until Matt took over.